pom.xml

```diff
@@ -377,11 +376,12 @@
           </configuration>
         </plugin>
         <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-deploy-plugin</artifactId>
-          <version>3.1.4</version>
+          <groupId>org.sonatype.central</groupId>
+          <artifactId>central-publishing-maven-plugin</artifactId>
+          <version>0.8.0</version>
+          <extensions>true</extensions>
           <configuration>
-            <skip>${gson.isInternalModule}</skip>
+            <publishingServerId>central</publishingServerId>
```

Comment:
I eventually concluded that [we did not need this](https://github.com/google/auto/pull/1953/files), and I've successfully released projects in that state.

But if you want to merge as-is because you know that things are working, I'm fine with that :)